third_party/ibis/ibis_teradata/api.py

```diff
@@ -49,11 +49,13 @@ def connect(
         A Database username to connect with
     password : str
         Password for supplied username
+    logmech : Optional[str]
+        Logmech flag to select with (default. TD2)
```

Comment:
Nit: Can we update this to reflect the order of the parameters? (logmech after port)

Reply:
yep!